vue-components/src/components/Popover.vue

```diff
@@ -179,5 +200,127 @@ $base: '.wikit-Popover';
 			transform-origin: bottom left;
 		}
 	}
+
+	&--bottom-center {
+		#{$base}__pointer {
+			inset-inline-start: calc(50% - #{$wikit-Popover-pointer-width} / 2);
+		}
+	}
+	&--bottom-right {
+		#{$base}__content-wrapper {
+			transform: translateX(0);
+		}
+		#{$base}__pointer {
+			inset-inline-start: $wikit-Popover-pointer-width / 2;
+		}
+	}
+	&--bottom-left {
+		#{$base}__content-wrapper {
+			transform: translateX(-100%);
+		}
+		#{$base}__pointer {
+			inset-inline-start: calc(100% - (#{$wikit-Popover-pointer-width} + #{$wikit-Popover-pointer-width} / 2));
+		}
+	}
+
+	&--top-center {
```

Comment:
I think this can just be `--top`, (and `--right`, etc.).

Reply:
(I know that it is stated differently in the task, but after looking at https://bootstrap-vue.org/docs/components/popover I agree that having just `top`, `right`, `bottom`, `left` for the center positions is good enough)